public/themes/gnome_color-scheme-dark.css

```diff
@@ -0,0 +1,58 @@
+/*
+ * name: GNOME
+ * color-scheme: dark
+ */
+
+:root {
+  --body-bg: #2d2d2d;
+  --cms-bg: var(--body-bg);
+  --text-color: #fff;
+  --link-color: #3472bd;
+  --quote-color: #3472bd;
+  --error-color: #ae151c;
+  --success-color: #0f3b71;
+  --highlight-color-bg: #353535;
+  --highlight-color: #fff;
+  --hr-border: 1px solid #1b1b1b;
+  --button-bg: #343434;
+  --code-bg: #292929;
+  --button-color: #ECEFF4;
+  --button-border: 1px solid #1b1b1b;
+  --button-focus-outline: 2px solid #1b1b1b;
+  --button-disabled-bg: #323232;
+  --button-disabled-color: #eceff4;
+  --button-disabled-border: #1b1b1b;
+  --button-danger-bg: #ae151c;
+  --button-danger-color: #fff;
+  --input-bg: #2d2d2d;
```

Comment:
From the posted screenshot for the login screen, I think this have to change. The input fields blend too much with the background and they just don't feel like an input field.

Reply:
Here is one idea:

![C96BC6B2-19E8-4541-8B65-3DF6A174860D](https://user-images.githubusercontent.com/45729/119514645-281fd280-bdb0-11eb-9b03-a6a412f0d3bb.jpeg)

Reply:
How does this look?

![Screenshot from 2021-05-26 00-59-59](https://user-images.githubusercontent.com/7016957/119605890-14d92b00-bde1-11eb-95ef-fa055e511a4c.png)

Reply:
I still don't think there's enough contrast, but I'll take it anyways. I think it's better to iterate later on 👍